README.md

```diff
@@ -2,27 +2,146 @@
 
 The official SurrealDB library for JavaScript.
 
-[![](https://img.shields.io/badge/status-beta-ff00bb.svg?style=flat-square)](https://github.com/surrealdb/surrealdb.js) [![](https://img.shields.io/badge/docs-view-44cc11.svg?style=flat-square)](https://surrealdb.com/docs/integration/libraries/javascript) [![](https://img.shields.io/badge/license-Apache_License_2.0-00bfff.svg?style=flat-square)](https://github.com/surrealdb/surrealdb.js)
+[![npm version](https://badge.fury.io/js/surrealdb.js.svg)](https://badge.fury.io/js/surrealdb.js)
+[![deno module](https://shield.deno.dev/x/surrealdb)](https://deno.land/x/surrealdb)
+
+[![](https://img.shields.io/badge/status-beta-ff00bb.svg?style=flat-square)](https://github.com/surrealdb/surrealdb.js)
+[![](https://img.shields.io/badge/docs-view-44cc11.svg?style=flat-square)](https://surrealdb.com/docs/integration/libraries/javascript)
+[![](https://img.shields.io/badge/license-Apache_License_2.0-00bfff.svg?style=flat-square)](https://github.com/surrealdb/surrealdb.js)
+
+## Quickstart-Guide
+
+### Install
+
+#### Deno
+
+Import it with
+
+```ts
+import Surreal from "https://deno.land/x/surrealdb/mod.ts";
+```
+
+> Note you should set a version in the url! For example
+> `https://deno.land/x/surrealdb@0.5.0/mod.ts`
+
+#### NodeJS, Browser with bundler
+
+For NodeJS or Browser with bundler like rollup, vite, webpack, ... you can just
+install the npm package:
+
+```sh
+npm i surrealdb.js
+# or with yarn
+yarn add surrealdb.js
+# or with pnpm
+pnpm i surrealdb.js
+```
+
+just import it with
+
+```ts
+import Surreal from "surrealdb.js";
+```
+
+or when you use commonjs
+
+```ts
+const Surreal = require("surrealdb.js");
```

Comment:
Small code fix:
```suggestion
const Surreal = require("surrealdb.js").default;
```

Reply:
This helped through my struggle to make this thing work. Thanks